nameko_grpc/connection.py

```diff
@@ -331,7 +394,19 @@ def send_request(self, request_headers):
         over the response.
 
         Invocations are queued and sent on the next iteration of the event loop.
+
+        raises ConnectionTerminatingError if connection is terminating. Check
+         connection .is_alive() before initiating send_request
+
+        Note:
+            We are handling termination and raising TerminatingError here as the
+            underlying library H2 doesn't do this. If H2 ever begins handling graceful
+            shutdowns, this logic will need altering.
```

Comment:
Is there a GitHub issue in H2 that can be referred to here? To make it easier to track if this ever does get implemented.